solutions/devsprint-viewcode-3/ViewCodeChallenge-RealEstate/Components/ButtonView.swift

```diff
@@ -0,0 +1,87 @@
+//
+//  ButtonView.swift
+//  ViewCodeChallenge-RealEstate
+//
+//  Created by Pedro Gabriel on 08/12/21.
+//
+
+import UIKit
+
+struct ButtonViewConfiguration {
+    let title: String
+}
+
+class ButtonView: UIView {
+    // MARK: Properties
+    private var button: UIButton = {
+        let button = UIButton(type: .system)
+        button.layer.cornerRadius = 10
+        button.backgroundColor = .systemBlue
+        button.titleLabel?.tintColor = .white
+        button.titleLabel?.font = .systemFont(ofSize: 18)
+        button.translatesAutoresizingMaskIntoConstraints = false
+        return button
+    }()
+    var pressedButton: (() -> Void)?
```

Comment:
Se `pressedButton` for configurado através do `configure`, ela poderá ser `private` aqui.